docs/docs/recipes.md

```diff
@@ -383,6 +383,76 @@ There are so many ways to add styles to your website; Gatsby supports almost eve
 - Use the CSS-in-JS library [Styled Components](/docs/styled-components/)
 - Use [CSS Modules](/tutorial/part-two/#css-modules)
 
+### Using Styled Components
+
+#### Prerequisites
+
+- A [Gatsby site](/docs/quick-start/) with an index page component
+- [gatsby-pluging-styled-components, styled-components, and babel-plugin-styled-components](/packages/gatsby-plugin-styled-components/) installed in `gatsby-config.js`
```

Comment:
I know this is unusual based on the other recipes we've shipped, but I think having a step to verify these are installed correctly would help people. Maybe just show a snippet of the gatsby-config file as Step # 1?

Reply:
I have added it as the first step in directions.